grafana/hatch.toml

```
@@ -0,0 +1,7 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["3.12"]
```

```suggestion
python = ["3.13"]
```
fix: we have upgraded to python 3.13